Sources/ArcGISToolkit/Extensions/SwiftUI/View.swift

```diff
@@ -152,3 +152,28 @@ extension View {
         }
     }
 }
+
+extension View {
+    /// Sets a closure to perform when a single tap occurs on the view.
+    ///
+    /// - Note: This is to retrofit the tap gesture to iOS 15.0.
+    /// - Parameters:
+    ///   - action: The closure to perform upon single tap.
+    ///   - screenPoint: The location of the tap in the view's coordinate space.
+    func onSingleTapGesture(perform action: @escaping (_ screenPoint: CGPoint) -> Void) -> some View {
```

Comment:
I have no problem if we just want to make the single tap functionality only available on iOS 16 and up. We are dropping support for iOS 15 after this April release.

Reply:
Since this extension is moved from the tabletop implementation, I'm OK with keeping the iOS 15 compatibility code, as it is not too verbose.